receiver/awscontainerinsightreceiver/go.sum

```diff
@@ -1041,6 +1041,11 @@ go.opencensus.io v0.23.0 h1:gqCw0LfLxScz8irSi8exQc7fyQ0fKQU/qnC/X8+V/1M=
 go.opencensus.io v0.23.0/go.mod h1:XItmlyltB5F7CS4xOC1DcqMoFqwtC6OG2xF7mCv7P7E=
 go.opentelemetry.io/collector v0.26.1-0.20210517234819-709d8a822e2e h1:jzx/BR04oreK33Mq4PfKgsZ9M7YGwVSU4NZ1A4BhOEY=
 go.opentelemetry.io/collector v0.26.1-0.20210517234819-709d8a822e2e/go.mod h1:J2oCzkvFAkgmgrvIdQNg5Dt3QAZ+ep7HNtHPay/7nvo=
+go.opentelemetry.io/otel v0.20.0 h1:eaP0Fqu7SXHwvjiqDq83zImeehOHX8doTvU9AwXON8g=
+go.opentelemetry.io/otel v0.20.0/go.mod h1:Y3ugLH2oa81t5QO+Lty+zXf8zC9L26ax4Nzoxm/dooo=
+go.opentelemetry.io/otel/metric v0.20.0/go.mod h1:598I5tYlH1vzBjn+BTuhzTCSb/9debfNp6R3s7Pr1eU=
+go.opentelemetry.io/otel/oteltest v0.20.0/go.mod h1:L7bgKf9ZB7qCwT9Up7i9/pn0PWIa9FqQ2IQ8LoxiGnw=
+go.opentelemetry.io/otel/trace v0.20.0/go.mod h1:6GjCW8zgDjwGHGa6GkyeB8+/5vjT16gUEi0Nf1iBdgw=
```

Comment:
What is inducing the dependency on the OTel Go API?  Should that be a dependency of the collector at this point?

Reply:
I think the dependency is coming from `github.com/open-telemetry/opentelemetry-collector-contrib/internal/aws/metrics`

<img width="954" alt="Screen Shot 2021-05-31 at 5 30 38 PM" src="https://user-images.githubusercontent.com/64551395/120251412-49f8d980-c236-11eb-93a0-468d16692f81.png">

Reply:
yea, AWS Metrics uses [attribute.Distinct](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/internal/aws/metrics/metric_calculator.go#L91) to do label dedup and sort